src/tests/test_network_profiler.py

```
@@ -0,0 +1,557 @@
+"""Unit tests for the NetworkProfiler module."""
+
+import threading
+import time
```

Ruff (0.12.2)

`time` imported but unused

Remove unused import: `time`

(F401)